src/components/Webinars/webinars.tsx

```diff
@@ -0,0 +1,32 @@
+import { getPaginatedPagesTotal, getPaginatedPosts, Pagination } from "@govtechsg/tradetrust-ui-components";
+import React from "react";
+import { FunctionComponent, useState } from "react";
+import { ResourceWebinar } from "../UI/ResourceWebinar";
+import { WebinarSingle } from "./types";
+
+export const webinarPerPage = 6;
+
```

Comment:
do we need to export for use elsewhere? if no need can put inside the component

Reply:
done.